Description: Fixes the compatibility with the version of the OSGi specification in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/bundles/org.eclipse.equinox.util/src/org/eclipse/equinox/internal/util/UtilActivator.java
+++ b/bundles/org.eclipse.equinox.util/src/org/eclipse/equinox/internal/util/UtilActivator.java
@@ -122,7 +122,7 @@
 				timeLog(33); /* 33 = "Creating Timer service took " */
 
 			int i = getInteger("equinox.util.threadpool.inactiveTime", 30);
-			timerReg = context.registerService(Timer.class, timer, null);
+			timerReg = context.registerService(Timer.class, (Timer) timer, null);
 			timer.addNotifyListener(ThreadPoolFactoryImpl.threadPool, Thread.NORM_PRIORITY, Timer.PERIODICAL_TIMER, (i * 1000L), 0);
 
 			TimerRef.timer = timer;
